docs/submitting.md

```diff
@@ -332,17 +332,21 @@ After submission:
 
 - An Associate Editor-in-Chief will carry out an initial check of your submission, and proceed to assign a handling editor.
 - The handling editor will assign two or more JOSS reviewers, and the review will be carried out in the [JOSS reviews repository](https://github.com/openjournals/joss-reviews).
-- Authors will respond to reviewer-raised issues (if any are raised) on the submission repository's issue tracker. Reviewer and editor contributions, like any other contributions, should be acknowledged in the repository.
+- Authors will respond to reviewer-raised issues (if any are raised) on the submission repository's issue tracker. Reviewer and editor contributions, like any other contributions, should be acknowledged in the repository. 
+- **JOSS reviews are iterative and conversational in nature.** Reviewers are encouraged to post comments/questions/suggestions in the review thread as they arise, and authors expected to respond in a timely fashion.
+- Authors and reviewers are asked to be patient when waiting for a response from an editor. Please allow two weeks for an editor to respond to a question before prompting them for further action.
```

Comment:
I feel like 2 weeks is really slow, unless an editor is out of the office.  I could imagine this leading our average review time to increase a lot, if editors decide that they can generally wait this long.